aw_server/custom_static.py

```diff
@@ -0,0 +1,69 @@
+"""
+Contains endpoints, as well as utility functions for custom static content.
```

Comment:
Throw an "experimental" notice here and I'd be willing to merge it :)

```suggestion
Contains endpoints, as well as utility functions for custom static content.

NOTE: Experimental, not (yet) implemented in aw-server-rust.
```